packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.Props.ts

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+export enum ReadingMode {
+  /**
+   * Do not read the content at all.
+   */
+  DoNotRead,
```

Comment:
This actually enable the developers to "turn off" the notification until he wants it to read again. It looks similar to how "aria-busy" works https://www.w3.org/TR/wai-aria-1.1/#aria-busy. It's true that this option will not be frequently used since the developer can control the string passed to the component anyway. Mine original idea is to provide this `good to have` feature as option.